src/utils/types/teachers.ts

```diff
@@ -0,0 +1,11 @@
+import { Teacher } from "./person";
+
+export interface TeachersList {
+  groupName: string;
+  content: Content[];
+}
+
+export interface Content {
```

Comment:
Naming this `Contact` doesn't tell much about what's it's containing so changing the name to something like `TeacherListContent` might be more suitable

P.S. to change the variable name in every file just hit `F2` key on vscode while highlighting the variable